### ISSUE TITLE
I have some problem when I use Skipping-The-Frame-Level

### ISSUE THREAD

**Issue by mashengtailang** [Yujia-Yan/Skipping-The-Frame-Level#5]
I get the following error on MacOS12.2.1 when installing transkun package:
mashengtailang@mashengtailangdeMacBook-Pro ~ % transkun input.mp3 output.mid
zsh: command not found: transkun

**Comment by Yujia-Yan** [comment]
Hi @mashengtailang
Can you provide more details on 1. how did you install the package? Was it successful? 2. Is the binary path included in your path? For example, if you are using homebrew version of pip, then it will be installed under /opt/homebrew/bin/ by default.

**Comment by mashengtailang** [comment]
Good morning!!
Happy to see your repeated!!
I'm a Musician but not a Program developer，And have a lot of things I don‘t understand. But I will try my best to answer you.

1. Yes I have installed the package and successed. 
2. I guess binary path is not included in my path. I just use MacOS's terminal to install/use these package.

**Comment by mashengtailang** [comment]
> Hi @mashengtailang Can you provide more details on 1. how did you install the package? Was it successful? 2. Is the binary path included in your path? For example, if you are using homebrew version of pip, then it will be installed under /opt/homebrew/bin/ by default.
Good morning!!
Happy to see your repeated!!
I'm a Musician but not a Program developer，And have a lot of things I don‘t understand. But I will try my best to answer you.

Yes I have installed the package and successed.
I guess binary path is not included in my path. I just use MacOS's terminal to install/use these package.

**Comment by Yujia-Yan** [comment]
It looks like the default path does not include the bin on Mac: Take a look at https://stackoverflow.com/a/62151306

**Comment by xb-li** [comment]
Hi @mashengtailang 
May I ask you a question?
When I was running the train file, I didn't know how to set runSeed, as shown in the following code:
`def train(workerId, nWorker, filename, runSeed, args):`
`    if num_processes == 1:
        train(0, 1, saved_filename)`
Thank you.

**Comment by cameronkkk** [comment]
Hi Yujia, Hope you are doing alright in NYC. 
This is the guy planned to run your great model on cellphone . After some try and error I'm able to run your great model
on my WIN10 PC. Every time I run it I got message below.
#1. Since I'm an AI newbie, Is there anything I should do with them?
================================================================================
"/home/cameron/.local/lib/python3.10/site-packages/torch/_dynamo/eval_frame.py:838: UserWarning: torch.utils.checkpoint: the use_reentrant parameter should be passed explicitly. In version 2.5 we will raise an exception if use_reentrant is not passed. use_reentrant=False is recommended, but if you need to preserve the current default behavior, you can pass use_reentrant=True. Refer to docs for more details on the differences between the two variants.
  return fn(*args, **kwargs)
/home/cameron/.local/lib/python3.10/site-packages/torch/utils/checkpoint.py:86: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("
===================================================================================

#2. I plan to buy a GPU and do some tweaks with AMT models. Do I need very high end GPU to tweak your great model?
     below is the message I found on (https://github.com/bytedance/piano_transcription)
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
All training steps are described in runme.sh. It worth looking into runme.sh to see how the piano transcription system is trained. In total 29 GB GPU memoroy is required with a batch size of 12. Users may consider to reduce the batch size, or use multiple GPU cards to train this system.

Results
The training uses a single Tesla-V100-PCIE-32GB card. The system is trained for 300k iterations for one week. 
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
#3.Can I ask Professor Duan questions about AMT as well?

Your time and reply would be very appreciated.

All the Best and Good Luck
